mix.exs

```diff
@@ -40,7 +40,7 @@ defmodule Paginator.Mixfile do
       {:ex_machina, "~> 2.1", only: :test},
       {:inch_ex, "~> 2.0", only: [:dev, :test]},
       {:postgrex, "~> 0.13", optional: true},
-      {:plug_crypto, "~> 1.2.0"}
+      {:plug_crypto, "~> 2.0.0"}
```

Comment:
I think, we could make it more permissive.
```suggestion
      {:plug_crypto, "~> 1.2.0 or ~> 2.0"}
```
As far as I can see in `plug_crypto` changelog https://github.com/elixir-plug/plug_crypto/blob/main/CHANGELOG.md

The API did not change between the two. The major version got bumped because of the internal algorithm change. However, the transition is smooth.

That means, it can work with both 1.2.x and 2.x.x.
We still keep the 1.2.0 minimum because it fixed an important bug.